sources/targets/Stride.Core.targets

```diff
@@ -30,7 +30,7 @@
       </ProjectReferenceWithConfiguration>
     </ItemGroup>
   </Target>
-
+  
```

Comment:
Could you remove this unnecessary change ?

Reply:
removed it